src/plan_expand_hypertable.c

```diff
@@ -1173,14 +1190,22 @@ ts_plan_expand_hypertable_chunks(Hypertable *ht, PlannerInfo *root, RelOptInfo *
 		propagate_join_quals(root, rel, &ctx);
 
 	inh_oids = get_chunk_oids(&ctx, root, rel, ht);
+
+	/* nothing to do here if we have no chunks and no data nodes */
+	if (list_length(inh_oids) + list_length(ht->data_nodes) == 0)
+		return;
+
 	oldrelation = table_open(parent_oid, NoLock);
 
 	/*
 	 * the simple_*_array structures have already been set, we need to add the
 	 * children to them. We include potential data node rels we might need to
 	 * create in case of a distributed hypertable.
 	 */
-	old_rel_array_len = root->simple_rel_array_size;
+#if PG12_GE
```

Comment:
Since this function exists also in PG12, it is unclear what this change has to do with PG13 support.

If this is a legitimate change for this PR, I would also consider implementing `expand_planner_arrays` for PG11 to get rid of the #if-#else.